documentation/modules/auxiliary/dos/cisco/CVE-2020-16139.md

```diff
@@ -1,27 +1,25 @@
 ## Vulnerable Application
 
-  1. Obtain a Cisco 7937G Conference Station.
-  2. Enable Web Access on the device (default configuration).
-  3. It has been observed that based on the firmware available from Cisco, all version are likely vulnerable.
+  [Cisco 7937G](https://www.cisco.com/c/en/us/support/collaboration-endpoints/unified-ip-conference-station-7937g/model.html) Conference Station.
+  This module has been tested successfully against SCCP-1-4-5-5 and SCCP-1-4-5-7.
```

Comment:
Again you still haven't included the vulnerable firmware versions here. Please include this in your description. Also is SCCP-1-4-5-7 a firmware version, a product, or something else? I can't tell from the description here. Same applies to the `## Scenarios` section.